docs/quickstart.md

```
@@ -27,76 +26,66 @@ cd flink-statefun
+  --bootstrap-server kafka:9092 \
+  --topic greeter.results \
+  --from-beginning --max-messages 1
+```
 
-    ```bash
-    docker logs -f statefun-remote-function
-    ```
+Expected output:
 
-    You should see a line like:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)